src/Microsoft.AspNet.Mvc.Core/Rendering/StringCollectionTextWriter.cs

```diff
@@ -22,6 +22,15 @@ public class StringCollectionTextWriter : TextWriter
         private readonly Encoding _encoding;
 
         /// <summary>
+        /// Creates a new instance of <see cref="StringCollectionTextWriter"/> with <see cref="Encoding"/>
+        /// defaulting to <see cref="UnicodeEncoding"/>.
+        /// </summary>
+        public StringCollectionTextWriter()
```

Comment:
sounds like much of this PR starts from the assumption we wanted a `StringWriter`  but we wanted a `TextWriter` with specific semantics, not a `StringWriter` in particular.